lib/platform/common.ts

```diff
@@ -129,6 +129,12 @@ export interface BranchStatusConfig {
   state: string | null;
   url?: string;
 }
+export interface FindPRConfig {
+  branchName: string;
+  prTitle?: string | null;
```

Comment:
@ViceIce doesn't `prTitle?` mean it can be undefined? Therefore it's now being defined as string or null or undefined?

Reply:
When strict mode is enabled something like `{prTitle: undefined}` is not allowed.

But currently strict is disabled, so that would be allowed.